README.md

```diff
@@ -1,2 +1,85 @@
-# shore-ruby-client
-Shore ruby client
+# Shore::Client
+
+Shore client to authorize API calls using [JWT](https://jwt.io)
+
+## Installation
+
+Add this line to your application's Gemfile:
+
+```ruby
+gem 'shore-client', github: 'shore-gmbh/shore-ruby-client', tag: 'v0.1.0'
+```
+
+And then execute:
+
+```sh
+$ bundle install
+```
+
+Or install it yourself as:
+
+```sh
+$ gem install shore-client
+```
+
+## Usage
+
+```ruby
+require 'shore/client'
```

Comment:
Is it 'shore/client' or 'shore-client'? If we could get it to be 'shore-client', then bundle will correctly require it. That would be nice. I always find it so annoying when gems have a different require than their name.